how-to/how-to-use-k8s-with-cri-containerd-and-kata.md

```diff
@@ -1,15 +1,15 @@
 # How to use Kata Containers and CRI (containerd plugin) with Kubernetes
 
-This document describes how to set up a single-machine Kubernetes cluster. 
-The Kubernetes cluster will use the CRI containerd plugin and Kata Containers to launch untrusted workloads.
+This document describes how to set up a single-machine Kubernetes cluster.
+The Kubernetes cluster will use the [CRI containerd plugin](https://github.com/containerd/cri) and [Kata Containers](https://github.com/kata-containers) to launch untrusted workloads.
```

Comment:
1. cluster _uses_ the
2. I'd recommend linking directly to the readme.md for the CRI plugin link, and Kata containers should link to: katacontainers.io or perhaps no link as we're already in a project hosted on github.com/kata-containers.